syntax/nickel.vim

```diff
@@ -73,9 +73,11 @@ syntax region nickelEnumTag start=+'"+ skip=+\\"+ end=+"+ oneline
 " Enum type
 syntax region nickelTypeContract start=+\[|+ end=+|]+ contains=nickelEnumTag
 
+syntax match nickelInterpolation "\v\%\{([^\}])*\}"
```

Comment:
For region that starts with an opening delimiter of two `%`, such as `m%%" ... "%%`, then interpolation is defined as `%%{`. I think we can use the same trick as below and just explicitly add syntax up to `n = 4` percent signs, by just duplicating the pattern:

```suggestion
syntax match nickelInterpolation1 "\v\%\{([^\}])*\}"
syntax match nickelInterpolation2 "\v\%%\{([^\}])*\}"
syntax match nickelInterpolation3 "\v\%%%\{([^\}])*\}"
syntax match nickelInterpolation4 "\v\%%%\{([^\}])*\}"
```